ncm-metaconfig/src/main/metaconfig/haproxy/main.tt

```diff
@@ -0,0 +1,67 @@
+global
```

Comment:
i would advise to make per section a separate tt file, and include it with indent like
```
global
[% INCLUDE 'metaconfig/haproxy.global.tt' FILTER indent %]
```
and in the `global.tt` you can avoid all indentation